sql/core/src/main/scala/org/apache/spark/sql/jdbc/JDBCRDD.scala

```diff
@@ -203,7 +203,8 @@ private[sql] object JDBCRDD extends Logging {
       fqTable: String,
       requiredColumns: Array[String],
       filters: Array[Filter],
-      parts: Array[Partition]): RDD[Row] = {
+      parts: Array[Partition],
+      fetchSize: Int): RDD[Row] = {
```

Comment:
ouch - is this an additional parameter to the existing constructor? What if there are other "hints" that can be provided to the SQL api, are we going to continuously add additional parameters? Better to opt for a Map of parameters, or at least going for a builder pattern. Other parameters that could be considered is the direction of the cursor (ResultSet.TYPE_FORWARD_ONLY), concurrency (ResultSet.CONCUR_READ_ONLY), result set direction (ResultSet.FETCH_FORWARD), etc.